### PR TITLE
Initial draft for the 1.9.0 release notes

### PR DESCRIPTION
**Pull request by nacx**
Please, comment and feel free to push anything that is missing directly to the branch in the ASF repo, to get the PR automatically updated.

### CODE REVIEW

**Comment by jclouds-commentator** [comment]
Go to http://953daad3023c62565e36-08c3ba73c884e16ca6b778d8da205806.r58.cf5.rackcdn.com/ to review your changes.

**Comment by jclouds-commentator** [comment]
Go to http://2e5cb29f9e51ce6f45ee-77c335958d07220d23cb7d91dd64540e.r78.cf5.rackcdn.com/ to review your changes.

**Comment by jclouds-commentator** [comment]
Go to http://37db3a251515c33db317-68a4243f54bcde19ba3fe74a7718fc7d.r85.cf5.rackcdn.com/ to review your changes.

File: _config.yml [hunk 1]
```diff
@@ -6,6 +6,6 @@ paginate: 5
 paginate_path: "blog/page:num"
 permalink: /blog/:categories/:year/:month/:day/:title
 relative_permalinks: false
-latest_version: 1.8.1
+latest_version: 1.9.0
 latest_version_date: 2014-10-25
```
Comment (demobox) [inline _config.yml]: I'm guessing this needs to be updated too?

File: releasenotes/1.9.0.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+1. [API Changes](#api)
+1. [Known Issues](#issues)
+1. [Credits](#credits)
+1. [Test Results](#test)
+
+## <a id="intro"></a>Introduction
+
+You can read the official announcement at [Apache jclouds 1.9.0 released](TODO). You can read the details of the specific JIRA issues addressed in this release at the [JIRA Release Notes](https://issues.apache.org/jira/secure/ReleaseNote.jspa?version=12329006&styleName=Html&projectId=12314430).
+
+To get jclouds, please see the [jclouds installation guide](/start/install/).
```
Comment (demobox) [inline releasenotes/1.9.0.md]: [minor] "To get started with jclouds"?

File: releasenotes/1.9.0.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+## <a id="highlights"></a>Highlights
+
+* Configured the Animal Sniffer plugin to make sure the code is runtime compatible with Java 6. This will help us support Android.
```
Comment (demobox) [inline releasenotes/1.9.0.md]: Link to Animal Sniffer?

File: releasenotes/1.9.0.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+* Added the Shipyard API
+* Added San José and Honolulu regions to CloudSigma
+* Improved Google Compute Engine Compute support
+* Chef has been promoted to the main repo and updated to support Chef 12
+* Improved Google Compute Engine to use AutoValue
+* Support portable setting and getting container and object ACLs (public-read and private)
+* Support portable multi-object delete
+* Improved multi-part upload compatibility with InputStream Payload
+* Add support for user metadata with filesystem provider
+* Add multi-part upload support for generic S3
```
Comment (demobox) [inline releasenotes/1.9.0.md]: Can we try to sort these into different categories to make them easier to read? E.g. "compute improvements", "blobstore improvements" etc. or so?

File: releasenotes/1.9.0.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+
+## <a id="api"></a>API Changes
+
+* Downgraded Guava dependency to 16.0.1 but code will now support also versions 17 and 18.
+* Removed all async interfaces from all apis and providers
+* Do not provide AWS endpoint for generic S3
```
Comment (demobox) [inline releasenotes/1.9.0.md]: "Generic S3 no longer defaults to an AWS endpoint"?

File: releasenotes/1.9.0.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+* Chef has been promoted to the main repo and updated to support Chef 12
+* Improved Google Compute Engine to use AutoValue
+* Support portable setting and getting container and object ACLs (public-read and private)
+* Support portable multi-object delete
+* Improved multi-part upload compatibility with InputStream Payload
+* Add support for user metadata with filesystem provider
+* Add multi-part upload support for generic S3
+
+
+## <a id="api"></a>API Changes
```
Comment (demobox) [inline releasenotes/1.9.0.md]: Should this be the first section, rather than "Highlights"?

File: releasenotes/1.9.0.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+* Do not provide AWS endpoint for generic S3
+* Require Azure API 2012-02-12 to allow support after [August 2015](http://blogs.msdn.com/b/windowsazurestorage/archive/2014/08/05/microsoft-azure-storage-service-version-removal.aspx)
+* Require Java 7 to use filesystem provider (enables user metadata support)
+
+TODO
+
+## <a id="issues"></a>Known Issues
+
+* Amazon S3 does not support signature v4 or Frankfurt
+* Filesystem provider does not support user metadata on Mac OS X due to [OpenJDK issue](https://bugs.openjdk.java.net/browse/JDK-8030048)
```
Comment (demobox) [inline releasenotes/1.9.0.md]: `[OpenJDK issue 8030048]`?

File: releasenotes/1.9.0.md [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+TODO
+
+## <a id="credits"></a>Credits
+
+jclouds would like to thank everyone who contributed time and effort in order to make this release happen:
+
+* Apache jclouds [PMC](http://people.apache.org/committers-by-project.html#jclouds-pmc) and community for verifying the release.
+* Check out who has been busy on [Open Hub](https://www.openhub.net/p/jclouds/contributors?query=&sort=latest_commit).
+
+TODO: Special thanks to the contributors from Google and Microsoft for their contributions in GCP and Azure.
```
Comment (demobox) [inline releasenotes/1.9.0.md]: Is the current sentence not enough? Perhaps link to the discussion threads on `dev@` announcing the contributions?

File: releasenotes/1.9.0.md (hunk 3 above)
Comment (andrewgaul) [inline releasenotes/1.9.0.md]: Done.

File: releasenotes/1.9.0.md (hunk 2 above)
Comment (andrewgaul) [inline releasenotes/1.9.0.md]: Done.

File: releasenotes/1.9.0.md (hunk 7 above)
Comment (andrewgaul) [inline releasenotes/1.9.0.md]: Done.

File: releasenotes/1.9.0.md (hunk 5 above)
Comment (andrewgaul) [inline releasenotes/1.9.0.md]: Done

File: releasenotes/1.9.0.md (hunk 4 above)
Comment (andrewgaul) [inline releasenotes/1.9.0.md]: Done.

File: releasenotes/1.9.0.md (hunk 6 above)
Comment (andrewgaul) [inline releasenotes/1.9.0.md]: Shouldn't we lead with the new features and not the plumbing like API changes?

**Comment by jclouds-commentator** [comment]
Go to http://70875139853e00180f66-873de8924bc43610585a990c029dd78c.r72.cf5.rackcdn.com/ to review your changes.

**Comment by andrewgaul** [comment]
Sorry, fighting with using the correct remote.

**Comment by jclouds-commentator** [comment]
Go to http://44c8eb9792aa142e969f-8bc165aad91a4057fd8d797a5f841c68.r48.cf5.rackcdn.com/ to review your changes.

File: releasenotes/1.9.0.md (hunk 6 above)
Comment (demobox) [inline releasenotes/1.9.0.md]: > Shouldn't we lead with the new features and not the plumbing like API changes?

Good question - I think it depends on who we expect to read release notes. If we think existing users are the most likely ones, breaking API changes that they need to be aware of seems like the most important topic? If new users, New and Cool stuff should come first, I agree...

File: releasenotes/1.9.0.md [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+
+## <a id="api"></a>API Changes
+
+* Downgraded Guava dependency to 16.0.1 but code will now support also versions 17 and 18.
+* Removed all async interfaces from all apis and providers
+* Generic S3 no longer defaults to an AWS endpoint
+* Require Azure API 2012-02-12 to allow support after [August 2015](http://blogs.msdn.com/b/windowsazurestorage/archive/2014/08/05/microsoft-azure-storage-service-version-removal.aspx)
+* Require Java 7 to use filesystem provider (enables user metadata support)
+
+TODO
```
Comment (demobox) [inline releasenotes/1.9.0.md]: Still needed?

File: releasenotes/1.9.0.md [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+
+## <a id="issues"></a>Known Issues
+
+* Amazon S3 does not support signature v4 or Frankfurt
+* Filesystem provider does not support user metadata on Mac OS X due to [OpenJDK issue 8030048](https://bugs.openjdk.java.net/browse/JDK-8030048)
+
+TODO
```
Comment (demobox) [inline releasenotes/1.9.0.md]: Still needed?

**Comment by demobox** [comment]
Thanks for the updates, @andrewgaul!

**Comment by jclouds-commentator** [comment]
Go to http://0815ae470e5a9b2997bc-fc90b1847c6c6ecad8ba9d148707cb21.r61.cf5.rackcdn.com/ to review your changes.

**Comment by jclouds-commentator** [comment]
Go to http://cfdc99d3a14de306aa84-f646b7349b1189c706b74e8d581c8166.r94.cf5.rackcdn.com/ to review your changes.

**Comment by nacx** [comment]
Merged and published. Thanks everyone!

**Comment by demobox** [comment]
Yaaaaay!! Many thanks, @nacx and everyone else who's worked on this!

**Comment by nacx** [comment]
I'm now uploading the javadocs... It's been uploading for about an hour now! *sigh*

**Comment by demobox** [comment]
> It's been uploading for about an hour now!

Perhaps we could offer to bind and ship them as a book, on demand? ;-)

**Pull request closed without merging**